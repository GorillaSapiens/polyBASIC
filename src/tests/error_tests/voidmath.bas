a = 5 + "void"
