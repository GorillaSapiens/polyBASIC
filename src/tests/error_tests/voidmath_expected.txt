ERROR: @1:5, OPERAND MISMATCH ❰INTEGER❱ ❰VOID❱
